### PR TITLE
DEL: AOPP

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -70,11 +70,6 @@
           <data android:scheme="bluewallet" />
           <data android:scheme="lapp" />
           <data android:scheme="blue" />
-          <data android:scheme="aopp" />
```
Comment (Overtorment) [inline android/app/src/main/AndroidManifest.xml:73]: only aopp, the rest we need for safello.
also, there should be ios counterpart i think

Comment (marcosrdz) [reply android/app/src/main/AndroidManifest.xml:73]: Fixed

File: screen/wallets/signVerify.js [hunk 2]
```diff
@@ -72,43 +68,15 @@ const SignVerify = () => {
     setLoading(true);
     await sleep(10); // wait for loading indicator to appear
     let newSignature;
-    const useSegwit = Boolean(params.aoppURI);
     try {
-      newSignature = wallet.signMessage(message, address, useSegwit);
+      newSignature = wallet.signMessage(message, address);
```
Comment (Overtorment) [inline screen/wallets/signVerify.js:72]: the default is true. is this what we want here? @limpbrains

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/f7hudgrwh0cuxfamfy19yu19w0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9452)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/chn31cwk7m3rmuccm7f88qb6gr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9453)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k9wk0dcbf1ub9a06qrbq8wvrgm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9454)

File: screen/wallets/signVerify.js (hunk 2 above)
Comment (limpbrains) [reply screen/wallets/signVerify.js]: Yes

Comment (limpbrains) [reply screen/wallets/signVerify.js]: Actually, no, I was looking at Outdated code.
It should use segwit and be `newSignature = wallet.signMessage(message, address);`

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: screen/wallets/signVerify.js (hunk 2 above)
Comment (marcosrdz) [reply screen/wallets/signVerify.js:72]: Done

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1j8fa4vhhqxrxu1xyec2rnyubw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9464)

**Review by limpbrains — APPROVED** [review]

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**